.azure-pipelines.yml

```diff
@@ -153,6 +153,14 @@ jobs:
         -n auto --max-worker-restart=0 -x \
         --junitxml=test-results-postgresql.xml
     displayName: 'Tests postgresql'
+  # The full disk test are run separately since they rely on running a sqlite3 database
+  # on a fuse mountpoint which can deadlock if run with the rest of the test suite
+  - bash: |
+      set -eux
+      py.test $(pytest.base_args) \
+        tests --rundiskfull --runslow -m diskfull -x \
+        --junitxml=test-results-memory.xml
```

Comment:
same junitxml file name than another test ? won't this erase the previous results ?

Reply:
Ooopsie, fixed!  82485f0